build/release/teamcity-mark-build.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+source "$(dirname "${0}")/teamcity-support.sh"
+
+tc_start_block "Variable Setup"
+build_label=$1
+build/builder.sh make .buildinfo/tag
+build_name="${TAG_NAME:-$(cat .buildinfo/tag)}"
```

Comment:
I just realized that the `TC_BUILD_BRANCH` variable can be used instead of `build_name` in the script. Once all uses of `build_name` are switched to `TC_BUILD_BRANCH`, these two lines can be deleted.

Reply:
This is possible because the value from `.buildinfo/tag` was used to create the git repo tag.